api/src/main/java/org/pmiops/workbench/db/dao/RdrExportDao.java

```diff
@@ -6,9 +6,19 @@
 import org.pmiops.workbench.db.model.DbRdrExport;
 import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 
 public interface RdrExportDao extends CrudRepository<DbRdrExport, Long> {
 
+  static final String WORKSPACE_IDS_TO_EXPORT_QUERY =
+      "select w.workspace_id from  workspace w LEFT JOIN "
+          + "rdr_export rdr on w.workspace_id = rdr.entity_id and rdr.entity_type = 2 where "
+          + "w.last_modified_time > rdr.last_export_date or rdr.entity_id IS NULL";
+  static final String UNCHANGED_WORKSPACE_IDS_QUERY =
+      "select entity_id from rdr_export where entity_id not in ("
```

Comment:
This probably also needs an `entity_type = 2` filter to be correct, or else reconstruct this SQL per my other comment